test/test_decomp.py

```diff
@@ -699,13 +709,14 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
             return real_out_unflat
 
     def check_decomposed(self, aten_name, mode):
-        self.assertTrue(
-            any(overload_to_aten_name(c) == aten_name for c in mode.decomposed),
-            msg=(f"aten.{aten_name} was not decomposed, saw calls for: "
-                 f"{', '.join(map(str, list(mode.called)))}. If your op is  "
-                 f"CompositeImplicitAutograd you should skip this test "
-                 f"by updating CROSS_REF_EXCLUDE_SET.")
-        )
+        if aten_name not in SKIP_CHECK_DECOMPOSED:
```

Comment:
@peterbell10 Previously we had avoided doing this, but since there is no `refs` implementation for `max` and `min` I couldn't figure out a better way to get around this. let me know if there are any other options.

Reply:
I could move these decomps to refs if that is your preference,but this problem will still persist. I think the issue is that for the max_binary test cases, calling e.g. torch.max will redirect to aten.maximum under the hood. So the test will see that nothing gets decomposed.

Reply:
From a conceptual standpoint though, I think a reference implementation of this op should be a one pass implementation. So my personal preference is to leave it as-is.

Reply:
> I could move these decomps to refs if that is your preference,but this problem will still persist.

Oh I see, the issue here is that  `max.other` and `max.out` are `CompositeImplicitAutograd` so the decomposition is never hit. In that case the fix should be to remove their decompositions.

> From a conceptual standpoint though, I think a reference implementation of this op should be a one pass implementation. So my personal preference is to leave it as-is.

The reference for `var_mean` decomposes into var + mean so I don't think there's any such rule.

Reply:
> Oh I see, the issue here is that max.other and max.out are CompositeImplicitAutograd so the decomposition is never hit. In that case the fix should be to remove their decompositions.

Even with the decompositions removed, `max.other` and `max.out` should be redirected to `maximum`, is that correct? I'm not familiar with what `CompositeImplicitAutograd` means, unfortunately.

Reply:
Yes, `CompositeImplicitAutograd` operators are transparent to the tracing mechanism so `maximum` will be traced directly without `ProxyTensor` ever getting dispatched for `max`.